lib/main.js

```diff
@@ -225,5 +225,9 @@ exports.onUnload = function (reason) {
         Services.search.currentEngine = Services.search.getEngineByName(ss.storage.last_default_engine);
     }
 
+    if (reason == 'uninstall') {
+        window.open('https://www.surveymonkey.com/r/KFL2J7T');
```

Comment:
I'm getting an error here:
```
JPM [error]   Message: ReferenceError: window is not defined
```

I think you need to add a require line for it first. 
https://developer.mozilla.org/en-US/Add-ons/SDK/High-Level_APIs/windows